week-2/extended-project/yoda-y-sus-apis/web_api.rb

```diff
@@ -0,0 +1,208 @@
+require_relative "web_api_methods"
+
+# Get information from APIs and Webs
+#=====================================================
+characters = get_data("https://swapi.co/api/people/")
+films = get_data("https://swapi.co/api/films/")
+planets =  get_data("https://swapi.co/api/planets/")
+species = get_data("https://swapi.co/api/species/")
+initialization_scrapping
+
+
+# Filter the data that we only need
+#=====================================================
+# characters
+#
+# name    : string
+# films   : array
+# gender  : string
+# species : array
+# id      : string
+
+characters_modified = characters.map do |value|
```

Comment:
You can probably just reuse the same variable to be honest:

```ruby
characters = characters.map do |value|
```

Unless you're using the original request values somewhere else, keeping the names meaningful matters more 😄